events/wc_class_events.txt

```diff
@@ -2717,16 +2739,86 @@ character_event = {
 }
 
 # summon_void_creature
-# Creates creature_nraqi
-# If FROM fails, kills FROM
+# Picks type of void creature
+character_event = {
+	id = WCCLS.1213
+	desc = EVTDESC_WCCLS_1213
+	picture = GFX_evt_nraqi
+	border = GFX_event_normal_frame_intrigue
+	
+	is_triggered_only = yes
+	
+	# C'thrax
+	option = {
+		trigger = {
+			religion = old_gods_worship
+			true_religion = old_gods_worship
+			OR = {
+				trait = being_void
+				is_9_10_physical_class_trigger = yes
+			}
+		}
+		name = EVTOPTA_WCCLS_1213
+		
+		set_character_flag = summon_void_creature_cthrax_flag
+		character_event = { id = WCCLS.1215 days = 3 }
+		
+		ai_chance = {
+			factor = 20
+		}
+	}
+	# N'raqi
+	option = {
+		trigger = {
+			religion = old_gods_worship
+			true_religion = old_gods_worship
+			NOT = {
+				trait = being_void
+				is_9_10_physical_class_trigger = yes
+			}
+		}
+		name = EVTOPTB_WCCLS_1213
+		
+		set_character_flag = summon_void_creature_nraqi_flag
+		character_event = { id = WCCLS.1215 days = 3 }
+		
+		ai_chance = {
+			factor = 20
+		}
+	}
+	# Merciless one
+	option = {
+		trigger = {
+			NOT = { has_character_modifier = pet_merciless_one }
+		}
+		name = EVTOPTC_WCCLS_1213
+		
+		clr_character_flag = summon_void_creature_ongoing_flag
+		
+		add_character_modifier = { name = pet_merciless_one years = 10 }
```

Comment:
It's weird that you summon zoatroids for only 10 years, but demons don't "expire"

Reply:
Changed!